src/Files.App/Views/Properties/GeneralPage.xaml

```diff
@@ -166,13 +166,13 @@
 							<RowDefinition Height="Auto" />
 						</Grid.RowDefinitions>
 
-						<!--  Item Path  -->
+						<!--  Item Location  -->
 						<TextBlock
 							Grid.Row="0"
 							Grid.Column="0"
 							HorizontalAlignment="Left"
 							Style="{StaticResource PropertyName}"
-							Text="{helpers:ResourceString Name=Path}"
+							Text="{helpers:ResourceString Name=Location}"
 							Visibility="{x:Bind ViewModel.ItemPathVisibility, Mode=OneWay}" />
```

Comment:
Can you update the property name as well so that it matches?

Reply:
This is what you meant right? I updated the Name=Location and the <!-- comment too

Reply:
It should be done in this PR to avoid confusion.

Reply:
> It should be done in this PR to avoid confusion.

Will get to it now.

Reply:
> Can you update the property name as well so that it matches?

I updated the `itemPathVisibility` to `itemLocationVisibility`. There is also a string `itemPath`. Do I update that to `itemLocation` as well or just the visibility?

Reply:
Yes Please.